CivicActions/govready-q
# ISPGBSS-554 Bug/control count

https://jiraent.cms.gov/browse/ISPGBSS-554

### Changes
- recalculating the way Controls Addressed is found (via data from the `controls_elementcontrol` table)
  - the total number of controls Addressed follows the baseline number (by entries matching the project id to the value in the `element_id` column)
  - the number of Controls Addressed out of total is the sum of "Assessed" and "Ready for assessment" (maps from integers in the `status` column)
- updated the percentage calculation for Controls Addressed

### Testing
_Note: This PR's changes work in combination with [this PR from the branding repo](https://github.com/CMSgov/rato-grq-branding/pull/26) and should be tested together_
- Visit the project page for a particular project and investigate the Controls Addressed section for the following:
  - Ensure that the **total** Controls Addressed is reflected by the project's number of baseline controls
  - Ensure that the **number out of the total** Controls Addressed is reflected by adding the project's "Assessed" and "Ready for assessment"
- Verify that the percent graphic displayed is reflective of the percentage of the satisfied Controls Addressed out of the total
